ci/jenkins/test-vmc.sh

```diff
@@ -179,6 +197,12 @@ function setup_cluster() {
     export OVA_TEMPLATE_NAME=${TEST_OS}-kube-${K8S_VERSION}
     rm -rf ${GIT_CHECKOUT_DIR}/jenkins || true
 
+    echo '=== Allocating IP ==='
+    trap release_static_ip Exit
```

Comment:
release_static_ip should not be added to trap, because test-vmc.sh already have a trap `trap cleanup_cluster EXIT`. Each bash script can only have one trap. Only the last registered trap is effective. We can remove this trap and invoke release_static_ip  in cleanup_cluster function.

I notice there are some left-over clusters in our lab. Later we shall manually cleanup them.